save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/test/TestsSourceVersionInfo.kt

```diff
@@ -6,19 +6,13 @@ import kotlinx.serialization.Serializable
 typealias TestsSourceVersionInfoList = List<TestsSourceVersionInfo>
 
 /**
- * @property organizationName
- * @property sourceName
+ * @property snapshotInfo
  * @property version
```

Comment:
Could you please document it

We have `commitId` which is actually `commitHash`, right? And what about `version` from now? What's the meaning of it

Reply:
`commitId` is `commitHash` -- an unique id of snapshot (tests)
`version` is human readable format provided by user: `tag`, `branch` or `short commit id`. 
We will support fetching by human readable format and creating a snapshot with `commitId` for internal usage and `version` to show for user to select for running a test

Will extend KDoc

Reply:
But, `version` from `TestSuitesSourceSnapshotKey` is still commit id, right?

Reply:
> But, `version` from `TestSuitesSourceSnapshotKey` is still commit id, right?

Not **still**: previously it was a version provided by user.
Now we will have two `TestSuitesSourceSnapshotKey`: one detected by preprocessor with `long commit id`, second -- with version provided by user

Reply:
Oh, I see, ok

Reply:
a note: it's a temporary till migration is finished.
We will remove `TestSuitesSourceSnapshotKey` and will use two entities:
- `TestsSourceSnapshot`: as a key to store in storage
- `TestsSourceVersion`: as a key\info for user. Will be stored only in database

Reply:
> Now we will have two `TestSuitesSourceSnapshotKey`

forgot to copy existed `TestSuites` -- added